jobs/scanning/images-health/Jenkinsfile

```diff
@@ -58,5 +56,32 @@ node() {
             currentBuild.result = "FAILURE"
             throw exception  // gets us a stack trace FWIW
         }
-    }
+
+        Date today = new Date();
+        Calendar c = Calendar.getInstance();
+        c.setTime(today);
+        if (c.get(Calendar.DAY_OF_WEEK) == 2 ) {  // to avoid spam message only check once on Mon
```

Comment:
Magic number `2` can be avoided by using constant like `Calendar.MONDAY`.

Also, how can I test the message if today is not Monday?

I would like to have a parameter to control whether to send the message, then move the weekday check to `scheduled_jobs/run_images-health` so we will have a central place to control how frequently this job should be triggered and how frequently we should send a message, without losing the flexibility to force send a message.